builtin/mainmenu/content/dlg_contentdb.lua

```diff
@@ -26,27 +26,21 @@ end
 -- Filter
 local search_string = ""
 local cur_page = 1
-local num_per_page = 5
-local filter_type = 1
-local filter_types_titles = {
-	fgettext("All packages"),
-	fgettext("Games"),
-	fgettext("Mods"),
-	fgettext("Texture packs"),
-}
+local filter_type
 
 -- Automatic package installation
 local auto_install_spec = nil
 
-local filter_types_type = {
-	nil,
-	"game",
-	"mod",
-	"txp",
+
+local filter_type_names = {
+	{ "type_all", nil },
+	{ "type_game", "game" },
+	{ "type_mod", "mod" },
+	{ "type_txp", "txp" },
 }
 
 
-local function install_or_update_package(this, package)
```

Comment:
unfortunate that this is global now. but it doesn't seem to be suitable for the `contentdb` namespace since that contains API stuff, not formspec stuff.